index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-screen">0</div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-AC">AC</button>
+      <button class="calculator-button calculator-positive-negative">+/-</button>
+      <button class="calculator-button calculator-percentage">%</button>
+      <button class="calculator-button calculator-divide">÷</button>
+    </div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-seven">7</button>
+      <button class="calculator-button calculator-eight">8</button>
+      <button class="calculator-button calculator-nine">9</button>
+      <button class="calculator-button calculator-multiply">×</button>
+    </div>
+    <div class="calculator-row">
+      <button class="calculator-button calculator-four">4</button>
+      <button class="calculator-button calculator-five">5</button>
+      <button class="calculator-button calculator-six">6</button>
+      <button class="calculator-button calculator-subtract">-</button>
```

Comment:
This is an awkward use of classes to identify buttons.  Using classes to differentiate button cattegories (numbers vs operators, etc.) is good, but for individual buttons like this it's a bit much.  It's probably better to use a data value, or simply grab the innerhtml to determine which specific button was pressed.